toolkit/committee-selection/authority-selection-inherents/src/authority_selection_inputs.rs

```diff
@@ -34,21 +41,31 @@ pub enum AuthoritySelectionInputsCreationError {
 			"Failed to get registered candidates for epoch: {0}, committee candidate address: {1}: {2}."
 		)
 	)]
+	/// Failed to get registered candidates for epoch
 	GetCandidatesQuery(McEpochNumber, String, Box<dyn std::error::Error + Send + Sync>),
 	#[cfg_attr(feature = "std", error("Failed to get epoch nonce for epoch: {0}: {1}."))]
+	/// Failed to get epoch nonce for epoch
 	GetEpochNonceQuery(McEpochNumber, Box<dyn std::error::Error + Send + Sync>),
 }
 
 #[derive(Debug, Clone, PartialEq, serde::Serialize)]
+/// Permissioned candidate data from Cardano main chain
 pub struct RawPermissionedCandidateData {
+	/// Unverified Partner Chain public key of permissioned candidate
 	pub sidechain_public_key: SidechainPublicKey,
+	/// Unverified Aura public key of permissioned candidate
 	pub aura_public_key: AuraPublicKey,
+	/// Unverified Grandpa public key of permissioned candidate
 	pub grandpa_public_key: GrandpaPublicKey,
 }
 
 #[derive(Debug, Clone, PartialEq, serde::Serialize)]
+/// Parameters for Ariadne selection algorithm.
```

Comment:
```suggestion
/// Subset of Ariadne selection algorithm parameters that is owned by the Partner Chain Governance Authority
```
We already have type that is described as "inputs". If inputs are anything different to parameters, then I would say that D-parameter is a parameter indeed, but permissioned candidates are input.